packages/@amazeelabs/silverback-search/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "@amazeelabs/silverback-search",
   "version": "1.3.4",
   "scripts": {
-    "test:unit": "cd ../../../apps/cms && vendor/phpunit/phpunit/phpunit web/modules/contrib/silverback_search"
+    "test:static": "cd ../../../apps/cms && pnpm test:static:phpcs ../../packages/@amazeelabs/silverback-search/drupal/ && pnpm test:static:phpstan ../../packages/@amazeelabs/silverback-search/drupal/",
```

Comment:
Can we provide central commands that automatically change directory to `apps/cms` without having to repeat it for each script command in each package?